src/utils/utils.go

```diff
@@ -230,3 +230,16 @@ func GenerateRequestContext() context.Context {
 func RequestIDLogger(ctx context.Context, log *logrus.Logger) logrus.FieldLogger {
 	return requestid.RequestIDLogger(log, requestid.FromContext(ctx))
 }
+
+func EtcdPatchRequired(openshiftVersion string) (bool, error) {
+	clusterVersion, err := version.NewVersion(openshiftVersion)
+	if err != nil {
+		return false, err
+	}
+	v47, err := version.NewVersion("4.7")
+	if err != nil {
+		return false, err
+	}
+
+	return clusterVersion.LessThan(v47), nil
```

Comment:
4.6.8 is less that 4.7? I assume go-version support it, right?

Reply:
Yup, added some more tests for these kinds of cases.